src/20_Components/amp-list.html

```diff
@@ -25,6 +25,38 @@
     bottom: 0;
     right: 0;
   }
+  <!--
+    #### CSS for the amp-list loader
```

Comment:
Great, now we have `The text (CDATA) inside tag 'style amp-custom' contains 'CSS i-amphtml- name prefix', which is disallowed.`.